webAO/master.js

```diff
@@ -64,9 +66,13 @@ export function check_https() {
 }
 
 export function setServ(ID) {
-	console.log(server_description[ID]);
-	if (server_description[ID] !== undefined) {
-		document.getElementById("serverdescription_content").innerHTML = "<b>" + online_counter[ID] + "</b><br>" + server_description[ID];
+	selectedServer = ID;
+
+	if (!lowMemory && document.getElementById(`server${ID}`).className === "")
+		checkOnline(ID, servers[ID].ip + ":" + servers[ID].port);
+
+	if (servers[ID].description !== undefined) {
+		document.getElementById("serverdescription_content").innerHTML = "<b>" + servers[ID].online + "</b><br>" + servers[ID].description;
```

Comment:
Neither `online` nor `description` are sanitized and can be used for XSS. We need to sanitize them.